supermarq-benchmarks/supermarq/plotting.py

```diff
@@ -205,8 +208,11 @@ def plot_benchmark(
     theta = radar_factory(num_spokes)
 
     _, ax = plt.subplots(dpi=150, subplot_kw=dict(projection="radar"))
+    assert isinstance(ax, RadarAxesMeta)
 
     title, labels, case_data = data
```

Comment:
definitely out of scope of this pr, but these should really be three different inputs..

Reply:
Went made this change in https://github.com/Infleqtion/client-superstaq/pull/1024 since it seemed like a quick minor fix